keps/sig-storage/1710-selinux-relabeling/README.md

```diff
@@ -668,10 +680,13 @@ No existing / new tests for volume mounting there.
 * Alpha of Phase 2 + 3:
   * Implemented `SELinuxChangePolicy` **with a separate alpha feature gate `SELinuxChangePolicy`** as preparation for `SELinuxMount` feature gate graduation.
   * Implemented SELinuxController.
-* Beta of Phase 2, alpha of phase 3:
+* Beta of Phase 2 + 3 (`SELinuxChangePolicy` is beta and enabled by default; `SELinuxMount` is beta, but disabled by default).
   * Telemetry numbers from OpenShift show that <5% of clusters would need to change any of their Pods.
-* GA:
+  * This phase signalizes that the feature is ready for real testing. Only non-breaking parts (`SELinuxChangePolicy`) are enabled by default.
+* GA of Phase 2 (`SELinuxChangePolicy` + `SELinuxMountReadWriteOncePod` are GA and locked to default):
   * All known issues fixed. Otherwise, we will GA Phase 1 only.
```

Comment:
Can you clarify which phase we will be in when this is implemented? Will - `SELinuxChangePolicy` will have an material effect beyond allowing users to opt-out of mount-option for applying selinux labels?

What happens if `SELinuxMount` is enabled (because we are moving this to beta), will `SELinuxChangePolicy` will then have an material impact?